libs/contract-utilities/tsconfig.lib.json

```diff
@@ -5,8 +5,7 @@
     "types": ["node"],
     "declaration": true,
     "composite": true,
-    "resolveJsonModule": true,
-    "declaration": true
```

Comment:
what does this remove?

Reply:
so i beleive this just removes a duplicate -- you can see the remaining `declaration` config above -- declarations are what generate the types through tsconfig and typescript